migrations/20200604110145_create_commit_table.ts

```diff
@@ -0,0 +1,20 @@
+import * as Knex from "knex";
+
+const commitTableName = "commit";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(
+    commitTableName,
+    (builder: Knex.TableBuilder) => {
+      builder.increments("id").primary();
+      builder.text("message").notNullable();
+      builder.timestamp("created_at").defaultTo(knex.fn.now());
```

Comment:
What's the meaning of this column?
- The creation time of this record in database?
- The commit time of the commit?
- The author time of the commit?

Reply:
I'll rename this to commit time